### PR TITLE
added support for releasing specific version

### PR DESCRIPTION
**Pull request by shahata**
now it is possible to `grunt release:1.2.3` in case you already know the version number that you would like to publish. unit test infrastructure needed to be enhanced in order to test this properly.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -62,16 +120,13 @@ module.exports = function(grunt) {
...
-    grunt.config.set('release.options.pushTags', false);
-    grunt.config.set('release.options.npm', false);
-    grunt.config.set('release.options.github', false);
-    grunt.config.set('release.options.additionalFiles', ['test/fixtures/_bower.json']);
+  });
+
+  grunt.registerMultiTask('releaseTest', function(){
+    var args = (this.data.args || []).join(':');
+    grunt.config.set('release', {});
+    grunt.config.merge({release: grunt.config.data[this.name]});
```
Comment (drublic) [inline Gruntfile.js]: Great merge.

**Comment by drublic** [comment]
Wow. Such good work. Thanks so much. :)

**Comment by shahata** [comment]
Thanks, I'm glad you like it :)
When do you think this can be merged?

**Comment by shahata** [comment]
@drublic any news about this?

**Comment by drublic** [comment]
Sorry for the delay. This is now part of master and will be released with the new version.
Thanks again for your work.

**Pull request closed without merging**